client.go

```diff
@@ -57,6 +57,7 @@ func NewIClient(creds Credentials, host string) (*IClient, error) {
 	c := &IClient{
 		Credentials: creds,
 		host:        host,
+		// Debug:		 true,
```

Comment:
Style: Please remove this commented line. 

Optional: if you could change the NewIClient method to take in a `debug` bool and set Debug: debug in the IClient instantiation, that would be a nice quality of life change!